assets/js/payment-method-extensions/payment-methods/stripe/credit-card/index.js

```diff
@@ -6,20 +6,22 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { stripePromise } from '../stripe-utils';
+import { loadStripe } from '../stripe-utils';
 import { StripeCreditCard } from './payment-method';
 import { PAYMENT_METHOD_NAME } from './constants';
 
 const EditPlaceHolder = () => <div>TODO: Card edit preview soon...</div>;
 
+const stripePromise = loadStripe();
+
 const stripeCcPaymentMethod = {
 	id: PAYMENT_METHOD_NAME,
 	label: (
 		<strong>
 			{ __( 'Credit/Debit Card', 'woo-gutenberg-products-block' ) }
 		</strong>
 	),
-	content: <StripeCreditCard />,
+	content: <StripeCreditCard stripe={ stripePromise } />,
```

Comment:
The `stripePromise` object is now a `unique` instance of `Stripe` for each stripe payment method (cc vs payment request). This is important to ensure there is no lingering "session" info between payment methods if they are switched. We keep the same promise for determining whether payment can be made or not though.